src/Splunk.OpenTelemetry.AutoInstrumentation/Metrics.cs

```diff
@@ -39,23 +40,16 @@ public MeterProviderBuilder ConfigureMeterProvider(MeterProviderBuilder builder)
 
     public void ConfigureMetricsOptions(OtlpExporterOptions options)
     {
-        if (!_settings.IsOtlpEndpointSet && _settings.Realm != null)
+        if (!_settings.IsOtlpEndpointSet && _settings.Realm != Constants.None)
         {
-            options.Endpoint = new Uri(string.Format(Constants.Ingest.MetricsIngestTemplate, _settings.Realm));
-        }
-
-        if (_settings.AccessToken != null)
-        {
-            string accessHeader = $"X-Sf-Token={_settings.AccessToken}";
-
-            if (string.IsNullOrEmpty(options.Headers))
+            if (string.IsNullOrEmpty(_settings.AccessToken))
             {
-                options.Headers = accessHeader;
-            }
-            else
-            {
-                options.Headers = $"{options.Headers}, {accessHeader}";
+                _log.Error($"'{ConfigurationKeys.Splunk.AccessToken}' is required when '{ConfigurationKeys.Splunk.Realm}' is set.");
```

Comment:
consider adding a unit test for this scenario

Reply:
Unit test added.